bin/webdriver-manager

```diff
@@ -103,6 +115,7 @@ var cli = optimist.
     describe('out_dir', 'Location to output/expect ').
     default('out_dir', SELENIUM_DIR).
     describe('seleniumPort', 'Optional port for the selenium standalone server').
+    describe('seleniumVersion', 'Optional selenium standalone server version').
```

Comment:
Can we have the `versions` object be available as a command line option, and use that for controlling all versions? So the invocation could look like:

```
> webdriver-manager update --versions.standalone=1.2.3 --versions.chromedriver=3.0
```